docs/meshcloud.project-resources.md

```diff
@@ -7,7 +7,11 @@ When a user is [assigned](meshcloud.project.md#assign-user-to-a-meshproject) to
 
 ## Project Dashboard
 
-In the project dashboard you get an overview of all locations and their platforms, that are available for for the selected meshProject. Via the **Manage** button on the card for a specific platform, you get to the [Project Platform Dashboard](#project-platform-dashboard).
+In the project dashboard you get an overview of all locations and their platforms, that are available for for the selected meshProject.
+Via the **Manage** button on the card for a specific platform, you get to the [Project Platform Dashboard](#project-platform-dashboard).
```

Comment:
I didn't change the above text. Just split to two lines to make it easier to read. Doesn't change the output in any way.